poolakey/src/main/java/ir/cafebazaar/poolakey/receiver/BillingReceiver.kt

```diff
@@ -0,0 +1,44 @@
+package ir.cafebazaar.poolakey.receiver
+
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+
+class BillingReceiver : BroadcastReceiver() {
+    override fun onReceive(context: Context?, intent: Intent?) {
```

Comment:
It would be nice to have a white space above this line.

Reply:
Done